### PR TITLE
release: 0.26.0

### PR DESCRIPTION
**Pull request by mistydemeo**
We're not quite ready to tag this - we have a few action items before then.

- [x] Ensure all docs for the new features are correct
- [x] Write docs for cross-compilation
- [x] Implement #1526.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,77 @@
...
+
+While we've always generated checksum information for binary tarballs/ZIPs, we only actually validated those checksums in the Homebrew installer. That changes with this release: we now embed checksum information into the shell installer and validate the tarball before unpacking it.
+
+- impl @fasterthanlime [Verify checksums in install.sh](https://github.com/axodotdev/cargo-dist/pull/1497)
+
+## cargo-auditable support
+
+We've added integrated support for the Rust Secure Code Working Group's [cargo-auditable](https://github.com/rust-secure-code/cargo-auditable), which embeds dependency information in your Rust binaries and makes it possible for users to check your binaries for the full dependency tree they were built from with their precise versions. For more information, see [our docs](https://opensource.axo.dev/cargo-dist/book/reference/config.html#cargo-auditable) and the [docs for the cargo-audit tool](https://github.com/rustsec/rustsec/blob/main/cargo-audit/README.md).
+
+(Note: for technical reasons, this feature and cross-compilation are currently mutually exclusive. Users can only enable one or the other.)
```
Comment (ashleygwilliams) [inline CHANGELOG.md:29]: this is interesting... should we file an issue to eliminate this?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,77 @@
...
+
+## cargo-cyclonedx support
+
+We've also added support for generating [CyloneDX Software Bill of Materials (SBOM)](https://cyclonedx.org) files for Rust projects. We've implemented this using the [cargo-cyclonedx](https://github.com/CycloneDX/cyclonedx-rust-cargo/blob/main/cargo-cyclonedx/README.md) tool. Unlike the cargo-auditable support above, which embeds dependency information directly into your binaries, this data is stored as a standalone `bom.xml` file which is distributed with your software. Users can then validate that SBOM file using [any compatible CycloneDX tool](https://cyclonedx.org/tool-center/).
+
+- impl @duckinator [Add cargo-cyclonedx config option.](https://github.com/axodotdev/cargo-dist/pull/1531)
+
+## omnibor support
+
+Rounding out this release's new security features, we've added support for [OmniBOR](https://omnibor.io) dependency tracking metadata for Rust projects. We implement this using the [omnibor-cli](https://github.com/omnibor/omnibor-rs/tree/main/omnibor-cli) tool. For more information, see [the docs](https://opensource.axo.dev/cargo-dist/book/reference/config.html#omnibor).
```
Comment (ashleygwilliams) [inline CHANGELOG.md:41]: so OmniBOR is not actually tracking dependencies so much as providing a way to identify or fingerprint the built software (which includes dependencies, but also includes first party code)

File: CHANGELOG.md (hunk 1 above)
Comment (mistydemeo) [reply CHANGELOG.md:29]: Unfortunately, I don't think we can. All three of `cargo-auditable`, `cargo-xwin` and `cargo-zigbuild` are implemented as cargo subcommands that wrap the real cargo builds. It's not possible to run more than one of them in the same build.

File: CHANGELOG.md (hunk 2 above)
Comment (mistydemeo) [reply CHANGELOG.md:41]: Oh, thanks for the clarification! I'll fix this.

File: CHANGELOG.md (hunk 1 above)
Comment (messense) [reply CHANGELOG.md:29]: IMO this is not strictly true, see https://github.com/rust-cross/cargo-zigbuild/issues/192

Comment (mistydemeo) [reply CHANGELOG.md:29]: Yep, been looking into it - #1608. Looking like the zigbuild stuff may be incompatible for other reasons, at least in testing so far.

**Review by ashleygwilliams — APPROVED** [review]

**Pull request merged**